lib/edge/kotlin/build-aar.sh

```
@@ -0,0 +1,194 @@
+fi
+
+if [ -z "${ANDROID_NDK_HOME:-}" ]; then
+    if [ -z "${ANDROID_HOME:-}" ]; then
+        echo "ERROR: ANDROID_NDK_HOME or ANDROID_HOME must be set." >&2
+        echo "  export ANDROID_NDK_HOME=/path/to/ndk" >&2
+        exit 1
+    fi
+    # Portable version sort (numeric by major.minor.patch): macOS/BSD `sort`
+    # has no `-V`, so `sort -V` would error here and break the documented
+    # ANDROID_HOME fallback on a Mac.
+    NDK_DIR=$(ls -d "$ANDROID_HOME/ndk/"* 2>/dev/null | sort -t. -k1,1n -k2,2n -k3,3n | tail -1 || true)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)